reconcile/slack_usergroups.py

```diff
@@ -237,6 +237,9 @@ def get_desired_state(slack_map, pagerduty_map):
             continue
         workspace = p['workspace']
         managed_usergroups = workspace['managedUsergroups']
+        skip_flag = p['skip']
```

Comment:
just a really stupid nit - if we're doing `continue` on this condition, we can probably move it two lines up to avoid the checking `workspace` and `managed_usergroups`